web/app/js/components/Top.jsx

```diff
@@ -144,13 +144,15 @@ class Top extends React.Component {
     });
   }
 
-  updateTapClosingState(isTapClosing) {
+  updateTapClosingState(isTapClosing, isTapRequestInProgress) {
     this.setState({
-      tapIsClosing: isTapClosing
+      tapIsClosing: isTapClosing,
+      tapRequestInProgress: isTapRequestInProgress
```

Comment:
This is a minor nit, but for consistency's sake I would switch the order of lines 149 and 150 to match the order of `handleTapStop`, to make it more readable later.